run_nwaku.sh

```diff
@@ -36,12 +36,12 @@ exec /usr/bin/wakunode\
       --rest-private=true\
       --rest-address=0.0.0.0\
       --rln-relay=true\
-      --rln-relay-dynamic=false\
+      --rln-relay-dynamic=true\
       --rln-relay-membership-index=${NODE_INDEX}\
```

Comment:
should the  --rln-relay-membership-index parameter be removed?
Can it instead be configured once the node is registered onchain?

Reply:
if the registered keystore only has 1 membership, it can be removed. if it has more than one, then you have to specific the index, since you have to specify which one to be loaded.

i would suggest removing it, since it would be easier to configure, and nodes are expected to have just one keystore with 1 membership.

Reply:
I removed it, for now the keystore will only have 1 membership